vhost-device-gpu/src/virtio_gpu.rs

```diff
@@ -281,7 +283,7 @@ impl AssociatedScanouts {
     }
 }
 
-#[derive(Default, Copy, Clone)]
```

Comment:
why removing `Copy` derive?

Reply:
I added the field `handle` needed for dmabuf scanout, and it does not implement copy.